SERVER/src/env_manager.py

```
@@ -0,0 +1,63 @@
+import os
+from .config import PROJECT_ROOT
+
+# Path to the .env file in the project root
+ENV_FILE_PATH = os.path.join(PROJECT_ROOT, ".env")
+
+def get_env_file_status():
```

Ruff (0.11.9)

Missing return type annotation for public function `get_env_file_status`

Add return type annotation: `str`

(ANN201)